1080i/Includes_OSD.xml

```diff
@@ -631,6 +631,19 @@
                 </control>
             </control>
 
+            <control type="image">
+                <bottom>150</bottom>
+                <right>view_pad</right>
+                <width>500</width>
+                <height>281</height>
+                <texture background="true">$VAR[Image_Player_Clearart]</texture>
+                <visible>![!String.IsEmpty(Window(Home).Property(OSDInfo)) | Player.ShowInfo | Window.IsActive(DialogFullScreenInfo.xml) | [Player.Paused + Skin.HasSetting(ShowInfoPaused)]]</visible>
+                <visible>!Window.IsVisible(DialogPlayerProcessInfo.xml)</visible>
+                <visible>Skin.HasSetting(ShowOSDClearArt)</visible>
+                <aspectratio>keep</aspectratio>
+                <animation effect="fade" start="0" end="100" delay="200" time="300">Visible</animation>
+                <include>Statusbar_Info_Clock</include>
```

Comment:
What is `<include>Statusbar_Info_Clock</include>` ??? 
I don't see this include defined anywhere.

Reply:
I included it by mistake, apparently I copied from a wrong file. It was from Aura.